src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimSpecCallbackHandler.java

```diff
@@ -248,25 +249,30 @@ private CallbackResponse setGenericResponseTypeFlag(CallbackParams callbackParam
             }
         }
 
+        if (caseData.getRespondent1ClaimResponseTypeForSpec() == RespondentResponseTypeSpec.PART_ADMISSION
+            || caseData.getRespondent1ClaimResponseTypeForSpec() == RespondentResponseTypeSpec.FULL_ADMISSION) {
+            updatedData.specFullAdmissionOrPartAdmission(YES).build();
```

Comment:
the calls to build in this if blocks are not needed. The resulting caseData is not put into a variable and in any case build it is called on updatedData when preparing the response of the method